src/lib/ext-activitylog.js

```diff
@@ -43,6 +43,10 @@ class View {
     this.saveLogBtn.addEventListener('click', this);
   }
 
+  setModelFilter(filterFunc) {
+    this.isFilterMatched = filterFunc;
```

Comment:
@Rob--W Should I give a different name this function instead of `isFilterMatched`? Does it make it hard to read since there is already a function named `isFilterMatched` in the controller?

Reply:
How about renaming it to `isModelFilterMatched` ?